packages/pointer_interceptor/pointer_interceptor/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: pointer_interceptor
 description: A widget to prevent clicks from being swallowed by underlying HtmlElementViews on the web.
 repository: https://github.com/flutter/packages/tree/main/packages/pointer_interceptor
```

Comment:
I think this needs to be updated to https://github.com/flutter/packages/tree/main/packages/pointer_interceptor/pointer_interceptor